subsys/bluetooth/mesh/shell/shell.c

```diff
@@ -138,9 +139,14 @@ static const struct bt_mesh_health_srv_cb health_srv_cb = {
 	.attn_on = attention_on,
 	.attn_off = attention_off,
 };
+#endif
+
+
 
 struct bt_mesh_health_srv bt_mesh_shell_health_srv = {
+#if defined(CONFIG_BT_MESH_SHELL_HEALTH_SRV)
```

Comment:
Isn't it? `health_srv_cb` won't be declared unless `CONFIG_BT_MESH_SHELL_HEALTH_SRV` is enabled, so I thought this was needed  to avoid compilation err.